packages/gatsby-remark-embed-snippet/README.md

```diff
@@ -164,7 +164,11 @@ function App() {
 
 ## How to use
 
-**Important**: This module must appear before `gatsby-remark-prismjs` in your plugins array, or the markup will have already been transformed into a code block and this plugin will fail to detect it and inline the file.
+**Important**: This module must appear before `gatsby-remark-prismjs` in your
+plugins array, or the markup will have already been transformed into a code
+block and this plugin will fail to detect it and inline the file.
+For further information about it's options, visit `gatsby-remark-prismjs`'s
```

Comment:
its

Reply:
\*miff\* I could pull the "non-native speaker card", but that one's too obvious \*facepalm\*
#corrected